src/test/java/io/moderne/organizations/OrganizationControllerTest.java

```diff
@@ -0,0 +1,103 @@
+package io.moderne.organizations;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.apache.commons.io.IOUtils;
+import java.nio.charset.StandardCharsets;
+import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.core.io.buffer.DataBuffer;
+import org.springframework.core.io.buffer.DefaultDataBufferFactory;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import io.moderne.organizations.OrganizationStructureService;
+import io.moderne.organizations.OrganizationController;
+import reactor.core.publisher.Flux;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PipedInputStream;
+import java.io.PipedOutputStream;
+
+@WebFluxTest(OrganizationController.class)
+public class OrganizationControllerTest {
+
+    @Autowired
+    private WebTestClient webTestClient;
+
+    @MockBean
+    private OrganizationStructureService organizationStructureService;
+
+    // Get content and throw a runtime exception if there is an error
+    private String getContent(InputStream stream) {
+        try{
+            return IOUtils.toString(stream, StandardCharsets.UTF_8);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
We usually use this more specific wrapper:
```suggestion
            throw new UncheckedIOException(e);
```